cpp/src/slice2cpp/Gen.cpp

```diff
@@ -414,12 +414,13 @@ namespace
             UnitPtr unt = p->container()->unit();
             string file = p->file();
             assert(!file.empty());
-            static const string prefix = "cpp:doxygen:include:";
             DefinitionContextPtr dc = unt->findDefinitionContext(file);
             assert(dc);
-            if (auto meta = dc->findMetadata(prefix))
+
+            // TODO: why do we ignore all instances of this metadata except the first?
```

Comment:
I guess because this is to document the required include for the type, and it should never be more than one header